aws/cloudformation/standalone/marketing/deployment/marketing.yml.erb

```diff
@@ -509,129 +468,171 @@ Resources:
+        Name: Marketing-Sites-Application-LB-VPC-Origin
+        OriginProtocolPolicy: http-only
 
-  MarketingGithubDeployerUser:
-    Type: AWS::IAM::User
+  CloudFrontOriginAccessControl:
+    Type: AWS::CloudFront::OriginAccessControl
     Properties:
-      UserName: marketing-github-deployer-development-cfn
+      OriginAccessControlConfig:
+        Name: !Sub "Cloudfront-OAC-${SubdomainName}.${BaseDomainName}"
+        SigningBehavior: always
+        SigningProtocol: sigv4
+        OriginAccessControlOriginType: s3
 
-  MarketingGithubDeployerUserPolicy:
-    Type: 'AWS::IAM::Policy'
-    Version: '2012-10-17'
+  StaticAssetsBucketCloudFrontPolicy:
+    Type: AWS::S3::BucketPolicy
     Properties:
-      PolicyName: MarketingGithubDeployerUserIamUserPolicy
-      Users:
-        - Ref: MarketingGithubDeployerUser
+      Bucket: !Ref StaticAssetsBucket
       PolicyDocument:
         Statement:
           - Effect: Allow
-            Action:
-              - 'cloudformation:CreateStack'
-              - 'cloudformation:DescribeStacks'
-              - 'cloudformation:CreateChangeSet'
-              - 'cloudformation:DescribeChangeSet'
-              - 'cloudformation:DeleteChangeSet'
-              - 'cloudformation:ExecuteChangeSet'
-            Resource: !Sub "arn:aws:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${AWS::StackName}/*"
-          - Effect: Allow
-            Action:
-              - 'iam:GetRole'
-              - 'iam:PassRole'
-            Resource:
-              Fn::GetAtt:
-                - FargateServiceTaskDefExecutionRole
-                - Arn
-          - Effect: Allow
-            Action:
-              - 'iam:GetRole'
-              - 'iam:PassRole'
-            Resource:
+            Principal:
+              Service: cloudfront.amazonaws.com
+            Action: s3:GetObject
+            Resource: !Sub '${StaticAssetsBucket.Arn}/*'
+            Condition:
+              StringEquals:
+                AWS:SourceArn: !Sub 'arn:aws:cloudfront::${AWS::AccountId}:distribution/${CloudFrontDistribution}'
+
+  ApplicationCachingPolicy:
+    Type: AWS::CloudFront::CachePolicy
+    Properties:
+      CachePolicyConfig:
+        Name: MarketingAppCachePolicy
+        DefaultTTL: 86400
+        MaxTTL: 31536000
+        MinTTL: 1
+        ParametersInCacheKeyAndForwardedToOrigin:
+          EnableAcceptEncodingGzip: true
+          EnableAcceptEncodingBrotli: true
+          HeadersConfig:
+            HeaderBehavior: whitelist
+            Headers:
+              - RSC
+              - Next-Router-State-Tree
+              - Next-Router-Prefetch
+              - next-url
+              - Accept
+              - x-prerender-revalidate
+          CookiesConfig:
+            CookieBehavior: whitelist
+            Cookies:
+              - __prerender_bypass
+          QueryStringsConfig:
+            QueryStringBehavior: none
+        Comment: Cache policy for the marketing app
+
+  CloudFrontDistribution:
```

Comment:
We discussed that you would create an SSL certificate for the cloudfront distribution, but given the limitation that ACM certificates for cloudfront can only be created in us-east-1, we should create a separate stack with the certificate resource, and an output of it's ARN. Maybe we name that stack `${StackName}-Cert`?

Reply:
I vote cert-specific, because it's not really shared.